x-pack/plugins/siem/public/containers/case/use_get_case_user_actions.tsx

```diff
@@ -48,6 +49,24 @@ export interface UseGetCaseUserActions extends CaseUserActionsState {
 
 const getExternalService = (value: string): CaseExternalService | null =>
   convertToCamelCase<CaseFullExternalService, CaseExternalService>(parseString(`${value}`));
+interface CommentsAndIndex {
+  commentId: string;
+  commentIndex: number;
+}
+// if the comment happens after the lastUpdateToCaseIndex, it should be included in commentsToUpdate
```

Comment:
@cnasikas here is your developer friendly comment, and i did move the reduce outside of the `caseUserActions.reduce` in a way ;)